packages/wagtail/modules/components/Blocks/BannerBlock/BannerDescriptionRichText/index.tsx

```
@@ -5,9 +5,7 @@ import { RichTextBlockWrapper } from './styled'
 const BannerDescriptionRichText = ({ value }: { value: string }) => (
   <RichTextBlockWrapper>
     {/* TODO (Tech Debt): Sanitize the HTML content with, for example, DOMPurify. */}
-    <div
-      dangerouslySetInnerHTML={{ __html: value }}
-    /> {/* eslint-disable-line react/no-danger */}
+    <div dangerouslySetInnerHTML={{ __html: value }} /> {/* eslint-disable-line react/no-danger */}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)